docs/topics/gradle/gradle-configure-project.md

```diff
@@ -136,20 +136,6 @@ file to:
 * `error` – the plugin will fail the build.
 * `ignore` – the plugin will skip the check and won't produce any messages.
 
-### Set custom JDK home
-
-By default, Kotlin compile tasks use the current Gradle JDK.
-If you need to change the JDK for some reason, you can set the JDK home with [Java toolchains](#gradle-java-toolchains-support)
-or the [Task DSL](#setting-jdk-version-with-the-task-dsl) to set a local JDK.
-
-> The `jdkHome` compiler option is deprecated since Kotlin 1.5.30.
```

Comment:
Should I return back this section and just throw this note inside it?

Reply:
No, this section should be removed